lib/internal/fs/streams.js

```diff
@@ -266,6 +302,9 @@ function WriteStream(path, options) {
   if (options.encoding)
     this.setDefaultEncoding(options.encoding);
 
+  if (this[kWrite] !== fs.write)
+    this._writev = undefined;
```

Comment:
We should be documenting this. Essentially it means that the stream will be slower if multiple chunks are waiting to be written.

Reply:
Updated docs. Better?

Reply:
Should we maybe allow passing `writev` as well? So that userland can do it's own optimization?

Reply:
> Should we maybe allow passing `writev` as well? So that userland can do it's own optimization?

I think you can feel free to do that, but in that case, you will have to describe the signature of the function because `fs` does not provide such a public API (that is, until https://github.com/nodejs/node/pull/25925 makes it way into core).